### PR TITLE
Updated the section for uninstalling Pipeline Operator.

### PR DESCRIPTION
**Pull request by LunguTheEngineer**
Jira issue : [RHDEVDOCS-5609](https://issues.redhat.com/browse/RHDEVDOCS-5609)

Aligned team: DevTools

OCP Version(s): 4.10 and later

Docs preview: [64067--docspreview](https://64067--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/uninstalling-pipelines)

SME review: @piyush-garg 

QE review: @ppitonak 

Peer review: @Srivaralakshmi

### CODE REVIEW

**Comment by piyush-garg** [comment]
the request was to mention `tekton-results` or `tekton-hub` in place of `tekton-chains`

**Review by ppitonak — APPROVED** [review]

File: cicd/pipelines/uninstalling-pipelines.adoc [hunk 1]
```diff
@@ -9,7 +9,7 @@ toc::[]
 Cluster administrators can uninstall the {pipelines-title} Operator by performing the following steps:
 
 . Delete the Custom Resources (CRs) that were added by default when you installed the {pipelines-title} Operator.
-. Delete the CRs of the optional components, such as {tekton-chains}, that are dependent on the Operator.
+. Delete the CRs of the optional components, such as {tekton-hub}, that are dependent on the Operator.
```
Comment (Srivaralakshmi) [inline cicd/pipelines/uninstalling-pipelines.adoc:12]: @masauso-lungu Is this component optional or not? Please clarify.

File: modules/op-deleting-the-pipelines-component-and-custom-resources.adoc [hunk 2]
```diff
@@ -28,5 +28,5 @@ Deleting the CRs will delete the {pipelines-title} components, and all the tasks
 
 [IMPORTANT]
 ====
-Repeat the procedure to find and remove CRs of optional components such as {tekton-chains}, before uninstalling the Operator. If you uninstall the Operator without removing the CRs of optional components, you cannot remove them later.
+Repeat the procedure to find and remove CRs of optional components such as {tekton-hub}, before uninstalling the Operator. If you uninstall the Operator without removing the CRs of optional components, you cannot remove them later.
```
Comment (Srivaralakshmi) [inline modules/op-deleting-the-pipelines-component-and-custom-resources.adoc:31]: @masauso-lungu Is this component optional or not? Please clarify.

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@masauso-lungu Please clarify my queries. Thanks!

File: modules/op-deleting-the-pipelines-component-and-custom-resources.adoc (hunk 2 above)
Comment (masauso-lungu) [reply modules/op-deleting-the-pipelines-component-and-custom-resources.adoc:31]: > the request was to mention `tekton-results` or `tekton-hub` in place of `tekton-chains`

Yes @Srivaralakshmi , that is one of the two options the SME told me to consider in place of `tekton-chains`

File: cicd/pipelines/uninstalling-pipelines.adoc (hunk 1 above)
Comment (Srivaralakshmi) [inline cicd/pipelines/uninstalling-pipelines.adoc:12]:
```suggestion
. Delete the CRs of the optional components such as {tekton-hub} that depend on the Operator.
```

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@masauso-lungu Please address the requested changes.

**Review by Srivaralakshmi — APPROVED** [review]
@masauso-lungu Approved!

**Comment by jeana-redhat** [comment]
Merge review looks good, but I see @mramendi listed as peer review but no comment from them yet. I am removing the merge labels, please add it back to the queue when you get that peer review ack (or lmk if this is unnecessary for devtools process).

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Comment by masauso-lungu** [comment]
> Merge review looks good, but I see @mramendi listed as peer review but no comment from them yet. I am removing the merge labels, please add it back to the queue when you get that peer review ack (or lmk if this is unnecessary for devtools process).
> 
> /remove-label merge-review-in-progress /remove-label merge-review-needed

Hello @jeana-redhat , Peer review was done by Sri, I have updated the request

**Comment by jeana-redhat** [comment]
@masauso-lungu thanks! sorry I am not super familiar with the folks on your team (yet :slightly_smiling_face:)

**Comment by jeana-redhat** [comment]
LGTM

/remove-label merge-review-needed

**Pull request merged**